backup_and_restore/control_plane_backup_and_restore/disaster_recovery/dr-hosted-cluster-within-aws-region.adoc

```diff
@@ -0,0 +1,128 @@
+:_content-type: ASSEMBLY
+[id="dr-hosted-cluster-within-aws-region"]
+= Disaster recovery for a hosted cluster within an AWS region
+include::_attributes/common-attributes.adoc[]
+:context: dr-hosted-cluster-within-aws-region
+
+toc::[]
+
+:FeatureName: Hosted control planes
+include::snippets/technology-preview.adoc[]
```

Comment:
I suggest moving your TP snippet after the abstract to avoid the TP notice from getting crawled by Google instead of your abstract.